tests/fps_tests.py

```diff
@@ -704,6 +706,184 @@ def test_expected_esld(self):
         self.assertEqual(loaded_sets, expected_sets)
         self.assertEqual(fp.error_list, [])
 
+class TestFindDiff(unittest.TestCase):
+    def test_unchanged_sets(self):
+        old_sets = {
+            'https://primary.com': 
+            FpsSet(
+                    primary="https://primary.com", 
+                    associated_sites=None,
+                    service_sites=None,
+                    ccTLDs={
+                        "https://primary.com": ["https://primary.ca"]
+                    }
+                    )
+        }
+        new_sets = {
+            'https://primary.com': 
+            FpsSet(
+                    primary="https://primary.com", 
+                    associated_sites=None,
+                    service_sites=None,
+                    ccTLDs={
+                        "https://primary.com": ["https://primary.ca"]
+                    }
+                    )
+        }
+        diff_sets, subtracted_sets = find_diff_sets(old_sets, new_sets)
+        self.assertEqual(diff_sets, {})
+        self.assertEqual(subtracted_sets, {})
+
+    def test_added_set(self):
+        old_sets = {
+            'https://primary.com': 
+            FpsSet(
+                    primary="https://primary.com", 
+                    associated_sites=None,
+                    service_sites=None,
+                    ccTLDs={
+                        "https://primary.com": ["https://primary.ca"]
+                    }
+                    )
+        }
+        new_sets = {
+            'https://primary.com': 
+            FpsSet(
+                    primary="https://primary.com", 
+                    associated_sites=None,
+                    service_sites=None,
+                    ccTLDs={
+                        "https://primary.com": ["https://primary.ca"]
+                    }
+                    ),
+            'https://primary2.com': 
+            FpsSet(
+                    primary="https://primary2.com", 
+                    associated_sites=None,
+                    service_sites=None,
+                    ccTLDs={
+                        "https://primary2.com": ["https://primary2.ca"]
+                    }
+                    )
+        }
+        expected_sets = {
+            'https://primary2.com': 
+            FpsSet(
+                    primary="https://primary2.com", 
+                    associated_sites=None,
+                    service_sites=None,
+                    ccTLDs={
+                        "https://primary2.com": ["https://primary2.ca"]
+                    }
+                    )
+        }
+        diff_sets, subtracted_sets = find_diff_sets(old_sets, new_sets)
+        self.assertEqual(diff_sets, expected_sets)
+        self.assertEqual(subtracted_sets, {})
+    def test_removed_set(self):
```

Comment:
```suggestion
        self.assertEqual(subtracted_sets, {})

    def test_removed_set(self):
```

Please put a blank line between different methods of a class. Same comment elsewhere. https://peps.python.org/pep-0008/#blank-lines